server/graphql/v2/mutation/AccountMutations.ts

```diff
@@ -48,6 +48,41 @@ const accountMutations = {
       });
     },
   },
+  addAccountTwoFactorAuth: {
+    type: new GraphQLNonNull(Account),
+    description: 'Add 2FA to the Account if it does not have it',
+    args: {
+      account: {
+        type: new GraphQLNonNull(AccountReferenceInput),
+        description: 'Account that will have 2FA added to it',
+      },
+      token: {
+        type: new GraphQLNonNull(GraphQLString),
+        description: 'The generated secret to save to the Account',
+      },
+    },
+    async resolve(_, args, req): Promise<object> {
+      if (!req.remoteUser) {
+        throw new Unauthorized();
+      }
+
+      const account = await fetchAccountWithReference(args.account);
+
+      if (!req.remoteUser.isAdminOfCollective(account)) {
+        throw new Forbidden();
+      }
+
+      const user = await models.User.findOne({ where: { id: account.CreatedByUserId } });
```

Comment:
We should not use `CreatedByUserId` for that, for some accounts this can be set to someone else (ie. if you create a profile to someone on their behalf). It's mostly for logging.

Also note that this `findOne` can return `null` if no user is associated with this account (for example, if you pass an organization profile rather than a user profile). 

```suggestion
      const user = await models.User.findOne({ where: { CollectiveId: account.id } });
```